### PR TITLE
DOCS-2229 API sessions

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/about-wallarm/api-sessions.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* [Identify performance issues](#analyzing-session-performance-issues) and bottlenecks to optimize user experience.
+* Know which API flow/business logic sequences will be affected before tuning a particular false positive, applying the virtual patch, adding rules, or enabling API Abuse controls.
+
+## How API Sessions module works
+
+The **API Sessions** module analyzes requests to your selected applications or their particular endpoints to join them into the user sessions and display these user sessions for security analysis.
+
+### Default rule
+
+By default, the module includes the default set of request parameters, basing on which values Wallarm will be able to join requests in the separate sessions. However, you always need to manually define the scope: applications and endpoints, for which the session discovery should be performed. You can also enable session discovery for all traffic.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-sessions.md:24]: what is the default set of parameters? which parameters are within?

File: docs/latest/about-wallarm/api-sessions.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+You can [get a list of shadow API endpoints](../user-guides/api-discovery.md#displaying-shadow-api) using the **API Discovery** module. Then, in **API Sessions**, use search to get the list of sessions with requests to these endpoints.
+
+## Analyzing session performance issues
+
+You can analyze session performance issues by expanding the session and then sorting its requests by the **Response time** column.
+
+## Enabling and configuring API Sessions
+
+The **API Discovery** module is disabled by default.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-sessions.md:58]: API Sessions module?

File: docs/latest/user-guides/api-sessions.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Configuring API Sessions
+
+By clicking the **Configure API Sessions** button in the **API Sessions** section, you proceed to the session discovery options:
+
+* At **Monitored sessions**, select applications or/and endpoints: only requests targeting them will be analyzed for the possibility of joining in a session. You can also select **Monitor all sessions**.
+
+    ![!API Sessions - Settings](../images/api-sessions/api-sessions-settings.png)
+
+* At **Traced parameters**, specify up to 10 request parameters which values should be exported to Wallarm in order to use them for the session identification. Use selected parameters in **Session identification rules**. If nothing is specified, the default set of parameters is used.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-sessions.md:21]: interesting to see the section contents

File: docs/latest/about-wallarm/api-sessions.md (hunk 1 above)
Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-sessions.md:24]: In the current version, we call it "built-in parameters/rules". My understanding is that we do not want to be more specific here. Asked the team ones again.

File: docs/latest/user-guides/api-sessions.md (hunk 3 above)
Comment (MaximBashurov) [reply docs/latest/user-guides/api-sessions.md:21]: Should we request a real-life-like example from the team?

File: docs/latest/about-wallarm/api-sessions.md (hunk 1 above)
Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-sessions.md]: And the answer is that built-in parameters/rules will be displayed in UI. So our mention of them in text is enough: we do not have to list or screenshot them as this set may change in time.

**Pull request closed without merging**